active_storage_base64.gemspec

```diff
@@ -1,15 +1,18 @@
 Gem::Specification.new do |s|
   s.name = 'active_storage_base64'
-  s.version = '0.1.0'
-  s.date = '2018-11-07'
-  s.summary = 'base64 support for ActiveStorage'
+  s.version = '0.1.1'
+  s.summary = 'Base64 support for ActiveStorage'
+  s.description = s.summary
+
+  s.files = Dir.chdir(File.expand_path('..', __FILE__)) do
+    'git ls-files -z'.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
 
-  s.files = [
-    'lib/active_storage_base64.rb'
-  ]
   s.require_paths = ['lib']
-  s.authors = ['Nicolas Fabre, Ricardo Cortio']
+  s.authors = ['Ricardo Cortio']
   s.license = 'MIT'
+  s.homepage = 'https://rubygems.org/gems/active_storage_base64'
```

Comment:
this should be `https://github.com/rootstrap/active-storage-base64`, right?

Reply:
🤦‍♂️ copied the wrong link